.travis.yml

```diff
@@ -4,6 +4,17 @@ node_js:
   - "0.10"
   - "0.12"
   - "iojs-v2.5.0"
+  - "4"
+  - "5"
+
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-4.8
+      - g++-4.8
```

Comment:
I wonder if it would be better to instead use the trusty image on Travis (https://docs.travis-ci.com/user/trusty-ci-environment/). Since it already has modern compilers it should speed up the CI time and also removes many lines from the .travis.yml file.

What do you think?